handler/rfc8628/storage.go

```diff
@@ -12,16 +12,15 @@ import (
 
 // RFC8628CoreStorage is the storage needed for the DeviceAuthHandler
 type RFC8628CoreStorage interface {
-	DeviceCodeStorage
-	UserCodeStorage
+	DeviceAuthStorage
 	oauth2.AccessTokenStorage
 	oauth2.RefreshTokenStorage
 }
 
-// DeviceCodeStorage handles the device_code storage
-type DeviceCodeStorage interface {
-	// CreateDeviceCodeSession stores the device request for a given device code.
-	CreateDeviceCodeSession(ctx context.Context, signature string, request fosite.Requester) (err error)
+// DeviceAuthStorage handles the device_code storage
```

Comment:
Change to `handles the device auth session storage` or `device_code & user_code storage`?